process/coordinator/transactionType.go

```diff
@@ -16,6 +17,10 @@ import (
 
 var _ process.TxTypeHandler = (*txTypeHandler)(nil)
 
+type relayedV3TransactionHandler interface {
+	GetInnerTransaction() *transaction.Transaction
```

Comment:
can't you add this function to the data.TransactionHandler interface ? and have GetInnerTransaction() data.TransactionHandler as the actual function ? It is better if we do not have concrete structure here.

Reply:
updated the TransactionHandler interface

Reply:
updated TransactionHandler interface